pkg/machines/machines.less

```diff
@@ -246,6 +246,16 @@ td > .utilization-bar-pf {
 // PF4 override: fix toggle expandable; it has a light grey background otherwise
 .pf-c-expandable__toggle {
     background: transparent;
+
```

Comment:
@garrett this need to be reported upstream as well and let's keep the issue link here in the code so that we can remove the custom css when fixed.

Reply:
Yeah, it makes sense. _However_, I don't see where in PF4 it's getting applied... but I do see the results. (It's kind of hard to see what does what in PF4, with variables referring to variables referring to variables.)

Reply:
This was because we were not importing a css file that is actually needed.
https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-core#pre-requisites
Imported the file ans removed this commit.

Reply:
Fixed by importing that base.css file from react-core.